specs/wp-signup-spec.js

```diff
@@ -38,6 +38,8 @@ import SideBarComponent from '../lib/components/sidebar-component';
 import * as SlackNotifier from '../lib/slack-notifier';
 
 import EmailClient from '../lib/email-client.js';
+import ThemesPage from '../lib/pages/themes-page';
+import ThemePage from '../lib/pages/theme-page';
```

Comment:
The file seems missing, should you use `lib/pages/theme-detail-page.js` and just add `pickThisDesign` to it?